automerge-example/standalone/rbac.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: demo-role-binding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: demo-role
+subjects:
+- kind: ServiceAccount
+  name: REPLACE_ME.YOUR_SA_NAME
+  namespace: REPLACE_ME.YOUR_NAMESPACE
+
+
+---
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: demo-role
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - services
+  - serviceaccounts
+  verbs:
+  - get
+  - list
+- apiGroups:
+  - apps
+  resources:
+  - deployments
+  verbs:
+  - get
+  - list
```

Comment:
This is an interesting one, I thought we needed tekton.dev permissions (for the SA to get the pipeline), because I initially saw my PipelineRun complaining with

```
    message: 'Error retrieving pipeline for pipelinerun openshift-pipelines/buildah-pipeline-run-2h58b:
      error when listing pipelines for pipelineRun buildah-pipeline-run-2h58b: etcdserver:
      request timed out'
    reason: CouldntGetPipeline
    status: "False"
```

but eventually everything worked - so.. a pleasant surprise? I am using the `openshift-pipelines` namespace, which may well have permissions applied at the namespace level (not SA) level... @dibbles not necessarily an issue, wondering if you tried with other namespaces that may not have the same privileges? 

That worked fine, onto the webhooks one now

Reply:
Pretty sure everything just worked for me - can't be sure which namespace I used, but I thought I used an alternate namespace called "foo"